core/templates/dev/head/components/skills-mastery-list/skills-mastery-list.constants.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Constants for the skills mastery list.
+ */
+
+angular.module('oppia').constant('MASTERY_CUTOFF', {
```

Comment:
With the latest develop, this should be split into `*constants.ts` and `*constants.ajs.ts` files. You can check some existing files like `topic-viewer-domain.constants.ts` and `topic-viewer-domain.constants.ajs.ts`.

Reply:
Done.